src/main/java/org/opentripplanner/transit/raptor/rangeraptor/standard/stoparrivals/view/StopsCursor.java

```diff
@@ -38,71 +38,117 @@ public StopsCursor(
         this.boardSlackProvider = boardSlackProvider;
     }
 
-    public boolean exist(int round, int stop) {
-        return stops.exist(round, stop);
+    public boolean reachedOnBoard(int round, int stop) {
+        var a =  stops.get(round, stop);
+        return a != null && a.reachedOnBoard();
     }
 
+    public boolean reachedOnStreet(int round, int stop) {
+        var a =  stops.get(round, stop);
+        if(a == null) { return false; }
+        return a.arrivedByAccessOnStreet() || a.arrivedByAccessOnStreet();
```

Comment:
Duplicated call to `arrivedByAccessOnStreet `

Reply:
As for now, this is only used in the debugger, so no "real" harm is done, but it is confusing - so very nice to get it fixed.